public/welcomestyle.css

```
@@ -0,0 +1,328 @@
+.maingrid{
+    display: grid;
+            grid-template-rows: 100px 700px;
+            position: relative;
+            height: 100vh;
+            width: 100%;
+            top: 150px;
+            margin-left: 100px;
+}
+.row1{
+    
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)